src/lib/tasks/run-texture-sync.ts

```diff
@@ -179,9 +180,10 @@ export function runTextureSync(coordinator: CoordinatorAPI): void {
     // The sprite was slated for removal. How to proceed depends on
     // whether it has more time left before its target arrival time.
 
-    if (properties.spriteView.TransitionTimeMs <= currentTimeMs) {
-      // The sprite was slated for removal, and its time has expired.
-      // Return its swatch for future reuse.
+    if (properties.zeroed &&
```

Comment:
The other part of fixing #77. Only 'remove' the sprite if it has had its values zeroed out first. Without this clause, a sufficiently short exit transition state would never be visible, and instead the removed sprite's swatch would have the old dirty values still in it. This would cause the last known state of the sprite to continue rendering.